scss/main.scss

```diff
@@ -1 +1,88 @@
 @use 'vars';
+
+// Track Circle Class
+@mixin circle($color, $size) {
+  background-color: $color;
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+  display: inline-block;
+}
+// Track Box Class
+@mixin box($color) {
+  border: 4px solid $color;
+  box-sizing: border-box;
+  padding: 10px;
+}
+
+// Track Circles
+.beginner-circle {
+  @include circle(#82ECEC, 183px);
+}
+
+.general-circle {
+  @include circle(#A88AA8, 183px);
+}
+
+.research-circle {
+  @include circle(#EA668E, 183px);
+}
+
+// Footer Circles
+.blue-circle {
+  @include circle(#82ECEC, 51px);
+}
+
+.purple-circle {
+  @include circle(#A88AA8, 51px);
+}
+
+.pink-circle {
+  @include circle(#EA668E, 51px);
+}
+
+// Track Boxes
+.beginner {
+  @include box(#82ECEC);
+}
+
+.general {
+  @include box(#A88AA8);
+}
+
+.research {
+  @include box(#EA668E);
+}
+
+.intro {
+  padding: 64px 32px;
+  background-color: #DED2E6;
+}
+
+.intro-header {
+  padding-top: 44px;
```

Comment:
nit: lets move this "padding-top" to the "outer-intro-layer" class just so all our padding for this section is nicely organized in one class